core/src/main/scala/kafka/server/KafkaBroker.scala

```diff
@@ -85,7 +85,8 @@ trait KafkaBroker extends KafkaMetricsGroup {
     explicitMetricName(Server.MetricsPrefix, KafkaBroker.MetricsTypeName, name, metricTags)
   }
 
-  newGauge("BrokerState", () => brokerState.value)
+  // visible for testing
+  private[server] val brokerStateGauge = newGauge("BrokerState", () => brokerState.value)
```

Comment:
Why do we need this versus the `brokerState` method?

Reply:
Agreed, we don't need it; `ServerStartupTest.testBrokerStateRunningAfterZK()` goes directly against the brokerState method.  Will fix.